index.js

```diff
@@ -48,15 +48,14 @@ const query = `
 let responseText = '[]';
 
 async function fetchRepos() {
-	const {body} = await got.post('https://api.github.com/graphql', {
-		json: true,
-		body: {query},
+	const {body} = await graphqlGot('https://api.github.com/graphql', {
```

Comment:
Maybe also support protocol-less? So we could just do `api.github.com/graphql`. Of course prepending HTTPS, not HTTP.

Reply:
Fixed.